packages/api/tests/es/test-stats.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const test = require('ava');
+const rewire = require('rewire');
+const get = require('lodash.get');
+
+const awsServices = require('@cumulus/aws-client/services');
+const s3 = require('@cumulus/aws-client/S3');
+const { randomString } = require('@cumulus/common/test-utils');
```

Comment:
It's you!  do you mind using randomId instead?  It really helps debug.  Only gotcha is for buckets on localstack they should be lowercase with no special characters.  e.g.`randomId('systembucket')`

Reply:
Yep maybe all my fault. I had copy/pasted. But I did fix.